lib/Translation/Escaper/Xliff12Escaper.php

```diff
@@ -116,6 +116,9 @@ private function parseInnerXml(string $content)
 
         $content = $node->asXml();
```

Comment:
please try to replace asXML with

$content = (string) $node

and remove your preg_replace again.

Let me know if this works.